lib/chef/provider/apt_repository.rb

```diff
@@ -331,8 +329,8 @@ def build_repo(uri, distribution, components, trusted, arch, add_src = false)
                    "[" + options.join(" ") + "]"
                  end
         info = [ optstr, uri, distribution, components ].compact.join(" ")
-        repo =  "deb      #{info}\n"
-        repo << "deb-src  #{info}\n" if add_src
+        repo =  "deb #{info}\n"
```

Comment:
This makes it much easier to read. We should just leave this

Reply:
> Yeah but were the quotes introduced to handle spaces in the URI:
> 
> ```
> deb      "http://deb.debian.org/debian with spaces" buster-updates main
> ```
> 
> Should that instead be:
> 
> ```
> deb      http://deb.debian.org/debian%20with%20spaces buster-updates main
> ```

sure @lamont-granquist will do thanks